telemetry/definitions/commonDefinitions.json

```diff
@@ -3403,43 +3381,16 @@
             ]
         },
         {
-            "name": "codeTransform_logApiError",
+            "name": "codeTransform_logApiResult",
             "passive": true,
-            "description": "A general logging measure for all client side API errors.",
+            "description": "A general logging measure for all client side API results.",
             "metadata": [
                 {
                     "type": "codeTransformApiErrorMessage",
-                    "required": true
-                },
-                {
-                    "type": "codeTransformApiNames",
-                    "required": true
-                },
-                {
-                    "type": "codeTransformJobId",
-                    "required": false
-                },
-                {
-                    "type": "codeTransformRequestId",
                     "required": false
                 },
                 {
-                    "type": "codeTransformSessionId",
-                    "required": true
-                },
-                {
-                    "type": "codeTransformTotalByteSize",
-                    "required": false
-                }
-            ]
-        },
-        {
-            "name": "codeTransform_logApiLatency",
-            "passive": true,
-            "description": "A general logging measure for all client side API's latency.",
```

Comment:
If you decide to keep these around, may want to change the description to:

    "description": "Deprecated. Use codeTransform_logApiResult instead.",